app/components/NestedCheckboxes.tsx

```diff
@@ -25,6 +25,15 @@ function allSame([first, ...rest]: any[]) {
   return rest.every((value) => value === first)
 }
 
+function isExternalLink(link: string): boolean {
+  try {
+    const url = new URL(link)
+    return url.hostname !== window.location.hostname
```

Comment:
So, this will only work on the client side. A simpler test is, if the URL starts with http:// or https://, then we should treat it as absolute.

Reply:
That makes sense,  what are your thoughts on this implementation:
```
function isExternalLink(link: string): boolean {
  try {
    return ['https:', 'http:'].includes(new URL(link).protocol)
  } catch (error) {
    return false
  }
}
```

Reply:
`new URL()` doesn't work on URLs without schemes, so this is more complicated than it needs to be. I suggest:

```ts
function isExternalLink(link: string): boolean {
  return link.startsWith('https://') || link.startsWith('http://')
}
```